lib/linter-rust.coffee

```diff
@@ -21,8 +21,9 @@ class LinterRust
       file = @initCmd do textEditor.getPath
       curDir = path.dirname file
       PATH = path.dirname @cmd[0]
-      options = JSON.parse JSON.stringify process.env
-      options.PATH = PATH + path.delimiter + options.PATH
+      options = new Object
```

Comment:
@Arcanemagus the only reason for this is my poor knowledge of CoffeeScript :) I'll fix this.